### DISCUSSION TITLE
Backwards compatibility

### DISCUSSION THREAD

**Discussion by tbffbt** [apohl79/audiogridder#708] [Q&A]
Hi,

first of all let me just say how much I appreciate this project, it has unbelievable potential. It has been an absolute game changer for my work, and the fact that it's free brings me hope in humanity 

I'd like to ask an honest question. How exactly do you approach OS backwards compatibility? I should say I didn't fully "get" AG until I successfully foraged the best version for my setup. Took a little while but I managed to find 2 versions up to this point. I'm still on Intel Mojave, on a 2018 Macbook Pro (in all its thermal throttling glory), still waiting for the best occasion to take the leap to M1, which entails quite a bit of adaptation I think. Is AG meant to be used with older OS's like mine or, given the ongoing development state of the project, is that something relegated to a more secondary role?

I'm asking because it seems to represent an even greater upgrade for those with less than optimal setups, like myself. Especially compared to what Apple Silicon is bringing to the table.

I've just tested the latest beta, which seems quite a leap in terms of features but unfortunately it lead me straight to that much dreaded constant CPU peaking, so I had to revert to the last stable version, the best one so far for me. Is that something to be worked on for the next stable version?

Anyway, thanks for your work, it's been helping out a lot of people. I even created a facebook group to help those looking for more information and the community has been getting stronger, hopefully even taking a load off your back.

easy,

Tomaz

**Comment by Fittkenny** [comment]
I have to say, with cpu peaking or not, what really works is having a fast gigabyte network with no glitches. It really has to be a stable network. I found the culprit in my case was an Asus router. Exchanged that for a gb switch and everything started working super smooth.

**Comment by tbffbt** [comment]
> I have to say, with cpu peaking or not, what really works is having a fast gigabyte network with no glitches. It really has to be a stable network. I found the culprit in my case was an Asus router. Exchanged that for a gb switch and everything started working super smooth.

I hear you, and thanks for that info!

I just forgot to mention the most important detail. I am using it as same server, I'm only using AG on this Macbook to take advantage of multi-threading in Ableton, which doesn't support it, and doesn't seem to have it in their list of priorities for the foreseeable future. It makes a world of a difference

**Comment by apohl79** [comment]
Hey I'm sorry for the silence, I'm busy with work and another project. Anyways, right now macOS is a bit of a pain, there are a bunch of builds when it comes to backwards compatibility, starting from 10.7, 10.8-11 for intel and 11 for arm/M1. the 10.8+ intel builds and the M1 builds also get combined into the universal binaries. So you can pick what you need.
